api/types.proto

```diff
@@ -447,8 +447,8 @@ message TaskStatus {
 // instructing Swarm on how this service should work on the particular
 // network.
 message NetworkAttachmentConfig {
-	// Target specifies the target network for attachment. This value may be a
-	// network name or identifier. Only identifiers are supported at this time.
+	// Target specifies the target network for attachment. This value must be a
+	// network ID.
```

Comment:
I don't think this is correct. The reason why this was called Target is for a possible move over in the future to use names or both name or ID. So I believe the previous comment should still be retained. But I am fine with removing `FindNetwork` since we don't expect a name with the current implementation.